### DISCUSSION TITLE
小米智能摄像机2 云台版 休眠状态和实际状态相反

### DISCUSSION THREAD

**Discussion by dabiaoge300** [XiaoMi/ha_xiaomi_home#205] [Q&A] [answered]
摄像头休眠设置对应的，【小米智能摄像机2 云台版 摄像机控制 开关】，这个状态和实际状态正好相反，开关关闭状态，【休眠设置】是打开状态，开关关闭【休眠设置】是启用状态

**Comment by SusanPhevos** [comment, marked as answer]
您好！请发一下设备详情页的截图，我们查一下是什么问题。

**Reply by dabiaoge300** [reply]
![IMG_1534](https://github.com/user-attachments/assets/2f53b0bf-6fe5-4584-8916-f1696141a037)
![IMG_1535](https://github.com/user-attachments/assets/5530a9b4-27f1-432f-8d4c-a4dd1e664430)
![微信截图_20241220101141](https://github.com/user-attachments/assets/3f32f59f-2109-4bfc-943d-a60327c41c48)
![微信截图_20241220101151](https://github.com/user-attachments/assets/5c2f8aab-5d76-431c-83bd-26bcc5cb172e)
在ha里面【小米智能摄像机2 云台版 摄像机控制 开关】这个功能对应的是米家只能摄像头休眠设置里的【休眠】功能，但是目前两个开关状态是相反的，这边开启那边是关闭，这边关闭那边是开启

**Reply by SusanPhevos** [reply]
设备厂商定义的[MIot-Spec-V2](https://home.miot-spec.com/spec/chuangmi.camera.039c01)该属性的名称没有“休眠”字样，因此xiaomi_home只能按照Spec原样显示厂商定义的名称。
